### ISSUE TITLE
Right ALT key sometimes doesn't work

### ISSUE THREAD

**Issue by brent-cx-developer** [wshanks/tbkeys#182]
I'm using Windows 10. Sometimes my keyboard layout is US (US Keyboard) while other times I switch to INTL (United States-International Keyboard). I've noticed that when I use the International layout, the right ALT key is no longer functioning as an ALT key. I'm told it's supposedly something called "AltGr". Unfortunately I have been able to set up mappings similar to "altgr+b". If I try that, it will be interpreted instead as simply "b" and trigger when I don't want it to. I've also tried "ctrl+alt+b" which also doesn't trigger when I use the right alt key in INTL layout.

TLDR: I'm trying to find a solution that will allow me to use the right alt key for an alt+b regardless of whether I'm in US or INTL layout, but nothing is working with tbkeys-lite

**Comment by wshanks** [comment]
I haven't done extensive research on this, but I think tbkeys might not be able to help here because AltGr is usually handled at a higher level of the operating system to change the keys into other characters (like changing e to an e with an accent). If you don't need that feature, maybe there is a way to change the layout to keep Alt instead of AltGr?

**Comment by eugenesvk** [comment]
The way it's handled by the OS is AltGr is identical to Ctrl+Alt (the OS injects Ctrl when you press RAlt), so strange that the combo doesn't work.

As a workaround you could set up Autohotkey to send right alt key combos when Thunderbird app is active even with your AltGr layout (or limit that to a few combos if you need AltGr for other keys)
